rules/do-you-use-loggermessage-in-net.md/rule.md

```diff
@@ -0,0 +1,114 @@
+---
+type: rule
+title: Do you use LoggerMessage in .NET?
+uri: do-you-use-loggermessage-in-net
+authors:
+  - title: Tom Iwainski
+    url: https://www.ssw.com.au/people/thomas-iwainski
+related: []
+created: 2024-11-05T12:52:08.000Z
+archivedreason: null
+guid: 2feb782f-0cee-4022-95d9-f44b9f739b90
+---
+
+Logging is a critical component in modern applications, but it can easily introduce performance overhead.
+The LoggerMessage class in .NET offers a solution to reduce the performance costs associated with logging. By utilizing this pattern, developers can optimize logging with fewer object allocations and reduced overhead.
+
+## Key Performance Benefits of LoggerMessage
```

Comment:
Hi @tomek-i 

- [ ] 1. Can you remove Capitalization from this and all other headings 2